dist/localforage.js

```diff
@@ -1521,7 +1525,7 @@ function _initStorage$1(options) {
 
         // Create our key/value table if it doesn't exist.
         dbInfo.db.transaction(function (t) {
-            t.executeSql('CREATE TABLE IF NOT EXISTS ' + dbInfo.storeName + ' (id INTEGER PRIMARY KEY, key unique, value)', [], function () {
+            t.executeSql(oneLine('CREATE TABLE IF NOT EXISTS ' + dbInfo.storeName + '\n                     (id INTEGER PRIMARY KEY, key unique, value)'), [], function () {
```

Comment:
We might want to avoid this call on every sqlite statement and keep it a single line to avoid any possible performance hit by running a regex.